data.sql

```diff
@@ -148,4 +149,105 @@ INSERT INTO animals (
         3,
         true,
         17
-        );
\ No newline at end of file
+        );
+
+
+--Insert the following data into the owners table:
+INSERT INTO owners (
+    full_name,
```

Comment:
- [ ] [Optional] I understand leaving a comment or two usually feels nice. But it is often considered good practice to take out all comments from the code base before merging, it would be nice to consider doing that before merging what do you think?